src/channels.rs

```diff
@@ -45,6 +45,16 @@ impl MiEI {
             .next()
     }
 
+    pub fn wildcard_roles<'a>(
+        &'a self,
+        wildcard: &'a str,
```

Comment:
You probably don't need this lifetime as `wildcard` is not returned, use `'_` instead.